src/documents/Sonification.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Sonification
+subheader:
+layout: default
+category: Methods
+---
+
+## What is Sonification?
+
+Sonification is a general term for conveying information or data through non-speech audio &mdash, similar to how the term "visualization" expresses the concept of conveying information or data graphically. Alert sounds for events such as receiving new email or program errors are simple forms of sonification commonly used in user interfaces for computers and other devices.
+
+*Data sonification* is an analogous audio technique to *data visualization* where structured data is converted into sound to try to:
+* communicate aspects of often-complex information clearly and efficiently
+* support specific analysis of aspects of the data
+* show patterns and relationships in the data
+
+## The Case for Sonification in Open Educational Resources
+
+While sonification is traditionally a theoretical and experimental area, it has rich potential as a means of supporting ["the creation of variations and enhancements"](/Techniques.html#AllowTheCreationOfVariationsAndEnhancements) in OER.
+
+Sonification's most obvious audience would seem to be the learner with visual impairments, but it can also be appreciated by those with cognitive impairments, learning disabilities, or a preference for audio presentation. It can help supplement data visualizations or open new areas for discussion, research and learning.
+
+### Floe Work
+
+The Floe team is experimenting with sonification and building authoring tools to support it as part of its work in exploring multimodal content creation and presentation in OERs.
+
+The <a href="http://build.fluidproject.org/chartAuthoring/demos/" target="_blank" class="link-external">Floe chart authoring tool</a> is a multimodal authoring tool that produces three forms of content from user-entered data data:
+
+* A pie chart (data visualization)
+* A data table that also functions as the pie chart's legend
+* A sonified form of the data (data sonification)
+
+### Sonification, Visualization and Underlying Data
+
+Placing sonification alongside the more familiar paradigm of data visualization explicitly surfaces that charts and graphs are a derived representation of a data set, and that one way OERs can support diverse learners and meet the [Accessibility Principles](/AccessiblityPrinciples.html) of perceivability and understandability is to make underlying data sets available wherever possible when presenting information in charts, graphs or other derived forms.
```

Comment:
Can we break up this sentence into smaller chunks? Perhaps break the sentence at the first comma and re-word the 2nd sentence a little.